tutorial-server/notebooks/CactusTutorial.ipynb

```diff
@@ -803,10 +859,99 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.plot(lin_data[:,1],lin_data[:,2]/lin_data[0,2], label=\"central density\")\n",
-    "plt.xlabel(r'$t$ [$M_{\\odot}$]');\n",
-    "plt.ylabel(r'$\\rho_c / \\rho_c(0)$');\n",
-    "plt.legend();"
+    "sim = SimDir(outdir)\n",
+    "# This sill print a list with all the data available\n",
+    "print(sim)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's plot the maximum pressure and compare it with the pressure as computed from the maximum density."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rho = sim.timeseries.maximum['rho']\n",
+    "press = sim.timeseries.maximum['press']\n",
+    "\n",
+    "# Polytropic constants\n",
+    "K, Gamma = 100, 2\n",
+    "\n",
+    "# Timeseries in kuibit support all the algebraic operations\n",
+    "cold_press = K * rho**Gamma\n",
+    "\n",
+    "plt.ylabel(\"Pressure\")\n",
+    "plt.xlabel(\"Time\")\n",
+    "plt.plot(cold_press, label=\"cold_press\")\n",
+    "plt.plot(press, label=\"press\", ls=\"dashed\")\n",
+    "plt.legend()\n",
+    "\n",
+    "max_diff = abs(press - cold_press).max()\n",
+    "\n",
+    "print(f\"Max difference {max_diff:.3e}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's plot an equatorial slice of the rest-mass density at the initial iteration."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "iteration_number = 0\n",
+    "rho_xy = sim.gridfunctions.xy[\"rho\"][iteration_number]\n",
+    "\n",
+    "# We cannot plot rho_xy directly because it contains all\n",
+    "# the information for the various refinement levels. \n",
+    "# We need to resample the data onto a uniform grid.\n",
+    "\n",
+    "# shape is the resolution at which we resample\n",
+    "# x0, x1 are the bottom left and top right coordiantes\n",
+    "# that we want to consider\n",
+    "\n",
+    "# Here we choose x0=[0,0] because we have reflection \n",
+    "# symmetry\n",
+    "\n",
+    "# resample=True activates multilinear resampling\n",
+    "\n",
+    "rho_xy_unif = rho_xy.to_UniformGridData(shape=[100, 100], \n",
+    "                                        x0=[0,0],\n",
+    "                                        x1=[10, 10],\n",
+    "                                        resample=True)\n",
+    "\n",
+    "# Undo reflection symmetry on the x axis\n",
+    "rho_xy_unif.reflection_symmetry_undo(dimension=0)\n",
+    "# Undo reflection symmetry on the y axis\n",
+    "rho_xy_unif.reflection_symmetry_undo(dimension=1)\n",
```

Comment:
this is already in the released version? Only rot180 was missing? The notebook should work with the ET release version and not require main of kuibit.

Reply:
These functions were introduced in version `1.3.0` (ET_2021_11 comes with `1.3.3`). Undoing rotational symmetry is not implemented yet.